### ISSUE TITLE
Add Caption/Transcripts to Audio and Video Elements

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#621]
The multimedia element does not have a captions track
`#video`
```
<video id="video" controls="controls" class="video-js vjs-default-skin" data-setup="{}" preload="auto">
    <source src="/downloads/6108vb28f?file=webm" type="video/webm">
    <source src="/downloads/6108vb28f?file=mp4" type="video/mp4">
    Your browser does not support the video tag.
  </video>
```

The multimedia element does not have an audio description track
`#video`

```
<video id="video" controls="controls" class="video-js vjs-default-skin" data-setup="{}" preload="auto">
    <source src="/downloads/6108vb28f?file=webm" type="video/webm">
    <source src="/downloads/6108vb28f?file=mp4" type="video/mp4">
    Your browser does not support the video tag.
  </video>
```

The multimedia element does not have a captions track
`#audio`
```
<audio id="audio" controls="controls" class="audiojs" preload="auto">
    <source src="/downloads/wm117p03j?file=ogg" type="audio/ogg">
    <source src="/downloads/wm117p03j?file=mp3" type="audio/mpeg">
    Your browser does not support the audio tag.
  </audio>
```

**Comment by jmcglone** [comment]
I would like to get Scott Williams input on how best to implement this with the existing data that we have.

**Comment by sethaj** [comment]
I think maybe AblePlayer https://github.com/ableplayer/ableplayer would be good. DCE used it in goldenseal, https://github.com/curationexperts/goldenseal. There's actually quite a bit in it's readme about accessibility referring to http://webaim.org/ and the DO-OT center at UW, http://www.washington.edu/doit/. 

They even used the accessibilty video Scott showed all of LIT *today* as one of their samples http://ableplayer.github.io/ableplayer/demos/video1.html which is kind of amazing.

I'll give it a try.

**Comment by sethaj** [comment]
Below is the caption file for "No Bees For Bridgeport [excerpt]". Add all of the text to the transcript field via /edit to get it to work.

[captions.vtt.txt](https://github.com/mlibrary/heliotrope/files/627475/captions.vtt.txt)

**Comment by sethaj** [comment]
I think we're going to add an extra field to the file_set model called "caption". It seems like we're in agreement that the transcript field isn't quite right for the caption (with timecode and machine readable headers).

Then we can experiment on possibly having the "transcipt" AblePlayer window open by default for assets that include a caption, somewhat like this example, http://wustl.curationexperts.com/concern/videos/hd76s0367 but without the TEI of course.

**Comment by sethaj** [comment]
Hold off on this until we have all the captions which is #636

**Comment by sethaj** [comment]
It looks like file_set already has a "caption", which is more of an image style caption. So if "transcript" isn't quite right for video captions, and "caption" is already used, we'll need to come up with a different field name to add to the FileSet model to hold machine-readable captioning information.

**Comment by sethaj** [comment]
Since file_set.caption already exists (it's for image captions) I'm going to stick with the transcript field. This will means that things in the transcript field should always be in WebVTT format. I'm trying to think of a use case where this wouldn't (or couldn't) be true and I can't really think of any.
